k2/python/k2/mutual_information.py

```diff
@@ -27,6 +27,7 @@ def forward(
         ctx,
         px: torch.Tensor,
         py: torch.Tensor,
```

Comment:
Oh, this function lacks of documentation about its arguments.

Reply:
Done, feel free to modify it if necessary.

Reply:
Thanks! Looks great to me.

Reply:
Could you fix the CI errors about the style issue?